posts/zzz_DO_NOT_EDIT_welcome,__a.../welcome,__ashley__tarasiewicz.qmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "Welcome, Ashley Tarasiewicz"
+author:
+  - name: Mike Stackhouse
+description: "Ashley Tarasiewicz joins the pharmaverse council as the Atorus Council representative."
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-04-15"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Community]
+# Feel free to change the image
+image: "ashley_tarasiewicz.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_welcome,__a..."
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+Hello pharmaverse community! 
+
+I’m thrilled to announce that Ashley Tarasiewicz will be taking over the Atorus seat from me on the Pharmaverse council. 
+
+Ashley has been a contributor to the pharmaverse since the early days of its inception, contributing to packages such as Tplyr, pharmaRTF, and metacore. One of Ashley’s biggest strengths is being a voice of the user, calling on her history working through clinical reporting and submissions. In the time since Ashley joined Atorus, she built Atorus Academy to help with the up-skilling of SAS programmers to learn R. Ashley has now transitioned to the product owner OpenVal, a validated R distribution. 
```

Comment:
```suggestion
Ashley has been a contributor to the pharmaverse since the early days of its inception, contributing to packages such as {Tplyr}, {pharmaRTF}, and {metacore}. One of Ashley’s biggest strengths is being a voice of the user, calling on her history working through clinical reporting and submissions. In the time since Ashley joined Atorus, she built Atorus Academy to help with the up-skilling of SAS programmers to learn R. Ashley has now transitioned to the product owner OpenVal, a validated R distribution. 
```

Reply:
looks good!  this should create links to the packages with our little link package :)